content/docs/lifting-state-up.md

```diff
@@ -9,24 +9,24 @@ redirect_from:
   - "docs/flux-todo-list.html"
 ---
 
-Often, several components need to reflect the same changing data. We recommend lifting the shared state up to their closest common ancestor. Let's see how this works in action.
+Seringkali, beberapa komponen perlu mencerminkan perubahan data yang sama. Kami menyarankan mengangkat `state` bersama ke modul utama terdekat. Mari kita lihat bagaimana ini bekerja dalam aksi.
 
-In this section, we will create a temperature calculator that calculates whether the water would boil at a given temperature.
+Di bagian ini, kami akan membuat kalkulator suhu yang menghitung apakah air akan mendidih pada suhu tertentu.
 
-We will start with a component called `BoilingVerdict`. It accepts the `celsius` temperature as a prop, and prints whether it is enough to boil the water:
+Kami akan memulai dengan komponen bernama `BoilingVerdict`. Ini menerima `celsius` suhu sebagai `props`, dan mencetak apakah itu cukup untuk merebus air:
```

Comment:
`We` di sini sepertinya mengacu pada `Kita`.


```
Ini menerima `celsius` suhu sebagai `props`, dan mencetak apakah itu cukup untuk merebus air:
```

sepertinya lebih cocok: 
```
Komponen tersebut akan menerima suhu `celsius` sebagai *prop*, dan menentukan apakah suhu cukup untuk membuat air mendidih
```

Reply:
Setuju dengan review mas @regalius, jangan lupa gunakan _props_, bukan _prop_ karena di https://github.com/reactjs/id.reactjs.org/wiki/Glosarium sudah ditentukan.

Reply:
Maaf, saya ingin meralat review saya, betul menggunakan _prop_ karena kalimat aslinya juga begitu. Yang perlu dirubah adalah penggunaan code block untuk _prop_. Gunakan _emphasis_ untuk term yang tidak bisa ditranslate, bukan `code block`.